### ISSUE TITLE
Synchronization scenario doesn't give confidence

### ISSUE THREAD

**Issue by evert-mouw** [keepassxreboot/keepassxc#7121]
## Overview

What will happen when one central database file (master) is synced (merged, copied) with multiple client (slave) databases, running on multiple client systems _simultaneously_?

## Steps to Reproduce

E.g. these database files are in use:

```
server:/export/home/ross/passwords.kdbx
yellow:/home/ross/passwords.kdbx
cyan:/home/ross/passwords.kdbx
```

Let's say on yellow and cyan we merge/sync each 5 minutes:

```
keepassxc-cli merge /home/ross/passwords.kdbx /mnt/server/home/ross/passwords.kdbx
cp /home/ross/passwords.kdbx /mnt/server/home/ross/passwords.kdbx
```

Also let's say both client computers (yellow and cyan) are active at the same time, and making these changes:

- yellow _adds_ a record
- cyan _changes_ a record

## Expected Behavior

It would be great if the addition and the change would replicate to both the server and the clients yellow and cyan.

## Actual Behavior

I didn't test but I'm afraid there is some risk of information loss.

## Question

How to do save and easy sync?

## Note

Current documentation on the website recommends to use syncing software, so the changed files would be copied without merging.

**Comment by droidmonkey** [comment]
Treat the file as a binary blob. You need to sync state with the blob before overwriting it or you will lose data. Guaranteed to lose data if you are doing raw file operations and not using a sync system that had version control. 

TLDR: Use syncthing.
